app/views/includes/register.jade

```diff
@@ -17,10 +17,15 @@
                     input#password.validate(type='password', name='password', required)
                     label(for='password') #{form.input.pass}
                         p.error.display-none.register-password-error
+                .input-field.col.s12
+                    input#checkbox_tos.validate(type='checkbox', name='checkboxTos',required)
+                    label#labelCheckTos.tosText(for='checkbox_tos') !{form.input.check_TOS}
```

Comment:
class names must not be "camel case", we are using "snake case": https://en.wikipedia.org/wiki/Snake_case